bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -420,148 +533,209 @@ Bridge netatmo:netatmoapi:home "Netatmo API" [ clientId="*********", clientSecre
+Number:Dimensionless Outdoor_Humidity                  "Humidity [%d %unit%]"                                       <humidity>         { channel = "netatmo:NAModule1:home:outside:Humidity" }
+Number               Outdoor_Humidex                   "Humidex [%.0f]"                                             <temperature_hot>  { channel = "netatmo:NAModule1:home:outside:Humidex" }
+Number:Temperature   Outdoor_HeatIndex                 "HeatIndex [%.1f %unit%]"                                    <temperature_hot>  { channel = "netatmo:NAModule1:home:outside:HeatIndex" }
+Number:Temperature   Outdoor_Dewpoint                  "Dewpoint [%.1f %unit%]"                                     <temperature_cold> { channel = "netatmo:NAModule1:home:outside:Dewpoint" }
+Number:Temperature   Outdoor_DewpointDepression        "DewpointDepression [%.1f %unit%]"                           <temperature_cold> { channel = "netatmo:NAModule1:home:outside:DewpointDepression" }
+Number               Outdoor_RfStatus                  "RfStatus [%.0f / 5]"                                        <signal>           { channel = "netatmo:NAModule1:home:outside:RfStatus" }
+Switch               Outdoor_LowBattery                "LowBattery [%s]"                                            <siren>            { channel = "netatmo:NAModule1:home:outside:LowBattery" }
+Number               Outdoor_BatteryVP                 "BatteryVP [%.0f %%]"                                        <battery>          { channel = "netatmo:NAModule1:home:outside:BatteryVP" }
+DateTime             Outdoor_TimeStamp                 "TimeStamp [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"                  <calendar>         { channel = "netatmo:NAModule1:home:outside:TimeStamp" }
+DateTime             Outdoor_LastMessage               "LastMessage [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"                <text>             { channel = "netatmo:NAModule1:home:outside:LastMessage" }
+
+# Rain Module
+Number:Length        Rain_Hour                         "Rain Last Hour [%.03f\"]"                                   <rain>             {channel="netatmo:NAModule3:home:rain:SumRain1"}
+Number:Length        Rain_Today                        "Rain Today [%.03f\"]"                                       <rain>             {channel="netatmo:NAModule3:home:rain:SumRain24"}
+Number:Length        Rain_Week                         "Rain This Week [%.03f\"]"                                   <rain>             {channel="netatmo:NAModule3:home:rain:SumRainThisWeek"}
+Number:Length        Rain_Month                        "Rain This Month [%.03f\"]"                                  <rain>             {channel="netatmo:NAModule3:home:rain:SumRainThisMonth"}
+Number               Rain_BatteryVP                    "Rain battery status [%d%%]"                                 <battery>          {channel="netatmo:NAModule3:home:rain:BatteryVP"}
```

Comment:
I am not sure that channel BatteryVP exists. Can you please check and fix the example.

Reply:
Yes it does exist.